tools/dependency/release_dates.py

```diff
@@ -38,32 +39,72 @@ def VerifyAndPrintLatestRelease(dep, repo, metadata_version, release_date):
   try:
     latest_release = repo.get_latest_release()
     if latest_release.created_at > release_date and latest_release.tag_name != metadata_version:
-      print(f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>: '
-            f'{latest_release.tag_name}@<{latest_release.created_at}>')
-  except github.UnknownObjectException:
+      print(Fore.YELLOW +
+            f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}> : '
+            f'{latest_release.tag_name}@<{latest_release.created_at}>' + Style.RESET_ALL)
+  # except github.UnknownObjectException:
+  except github.GithubException as e:
+    # print(Fore.RED + 'GitHub API status get_latest_release(): {}'.format(str(e)) + Style.RESET_ALL)
     pass
 
 
 # Print GitHub release date, throw ReleaseDateError on mismatch with metadata release date.
 def VerifyAndPrintReleaseDate(dep, github_release_date, metadata_release_date):
   mismatch = ''
   iso_release_date = FormatUtcDate(github_release_date)
-  print(f'{dep} has a GitHub release date {iso_release_date}')
+  print(Fore.GREEN + f'{dep} has a GitHub release date {iso_release_date}' + Style.RESET_ALL)
```

Comment:
Can you define a `def Colorize(color, str):` method that wraps up this pattern? Then we don't need to worry about missing `RESET_ALL`.

Reply:
Done